hoomd/filter/custom.py

```diff
@@ -10,7 +10,7 @@ class CustomFilter(Hashable, Callable):
     """Abstract base class for custom particle filters.
```

Comment:
In the example for this class, the class name should be `MassFilter`, not `DiameterFilter`

Reply:
Fixed.